.circleci/config.yml

```diff
@@ -0,0 +1,89 @@
+---
+version: 2.1
+
+docker_auth: &docker_auth
+  username: $DOCKERHUB_USERNAME
+  password: $DOCKERHUB_PASSWORD
+
+defaults: &defaults
+  docker:
+    - image: greenpeaceinternational/circleci-base:latest
+      auth:
+        <<: *docker_auth
+  working_directory: /home/circleci/
+jobs:
+  clone-tokens-repo:
+    <<: *defaults
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Clone planet4-design-tokens
+          command: |
+            mkdir tmp && cd "$_"
+            git clone -b PLANET-7002_implement-ci-automation https://github.com/greenpeace/planet4-design-tokens
```

Comment:
The branch `PLANET-7002_implement-ci-automation` is temporary added for a testing purpose. We'll use always the `main` branch.